PYME/DSView/modules/coloc.py

```diff
@@ -316,8 +316,8 @@ def OnColoc(self, event=None, restrict_z=False, coloc_vs_z = False):
         
         im.mdh['Colocalisation.Channels'] = names
         im.mdh['Colocalisation.Thresholds'] = [tA, tB]
-        im.mdh['Colocalisation.Pearson'] = pearson
-        im.mdh['Colocalisation.Manders'] = [MA, MB]
+        im.mdh['Colocalisation.Pearson'] = np.array(pearson)
```

Comment:
these should be scalars, casting them to an array will cause problems with serialisation. If casts are needed, they should be to `float`.

Reply:
Definitely needs a cast. Will switch to `float`.